MAINTAINING.md

```diff
@@ -21,6 +21,22 @@ If the pull request looks good but does not follow the commit conventions, use t
 
 ![Screenshot of GitHub's Squash and Merge confirm dialog](assets/squash-and-merge-dialog.png)]
 
+## Breaking changes
+
+When merging a breaking change using the PR body as specified above, extra care must be taken. Breaking changes must first be merged into the `beta` branch, where further testing may be conducted and additional breaking changes may be combined before cutting a release. Note the repository for the change: if it's dependent on other repos where the same change must be made, merge the leaf nodes first before the nodes higher up the tree. Your merge order should look something like:
+
+1. endpoint
+1. request
+1. plugins
+1. auth strategies
+1. core
+1. \*-methods
+1. oauth-app
+1. webhooks
+1. app
+1. octokit/octokit.js
+1. octokit/rest.js
+
```

Comment:
Do we (or should we) have a procedure for how often we diff the `beta` branch with `main` and create new releases?

Reply:
Usually the two shouldn't diverge, `beta` branches should be short lived. Or is your question about something else?

Reply:
I don't think there is a procedure, however one should create a draft PR for the next major version merging `beta` into `main`, ex: https://github.com/octokit/request.js/pull/586 and then mark it ready for review when it is ready

Reply:
In that case, should I amend the procedure to contain something like: 

"After the breaking changes are combined and tested in the `beta` branch, merge the `beta` branch into `main` using a PR with "BREAKING CHANGE" in the body as documented above. `beta` branches are intended to be short-lived."

Reply:
I'ved added similar language to the PR; please let me know what you think!

Reply:
Good pony by @wolfy1339, we should create beta pull requests as drafts until all planned breaking changes have been merged into it

Reply:
Maybe add that

1. beta branches should be deleted first and re-created based on main
2. Once the first pull request is merged into beta, create a draft pull request from beta to main with the title vX where X is the next breaking version.
3 Once all breaking changes are merged into beta, test and set the pull request to ready for review